packages/twenty-server/src/engine/twenty-orm/decorators/workspace-index.decorator.ts

```diff
@@ -25,7 +25,9 @@ export function WorkspaceIndex(
     );
 
     metadataArgsStorage.addIndexes({
-      name: `IDX_${generateDeterministicIndexName([
+      name: `IDX_${
```

Comment:
Not sure if we want that. I added IDX_UNIQUE so that we could differentiate field-level unicity constraints from the indexes that are managed directly. It probably doesn't matter a lot in either case